### DISCUSSION TITLE
Is there documentation on how to add a custom storage backend?

### DISCUSSION THREAD

**Discussion by bendavis78** [jgraph/drawio#2561] [Q&A]
I would like to create a custom storage backend, but am having trouble finding examples or where to start. Could someone point me in the right direction?

**Comment by davidjgraph** [comment]
What is the storage?

**Comment by gnaegi** [comment]
This is also my question: we would love to integrage draw.io in our OSS Learning Management System. For the integration of OnlyOffic and Collabora for example we could implement the WOPI standard which basically is a communication layer between the storage (our LMS) and the editor. During launch of the editor an URL is provided where the editor can pull the document and put it back when editing is finished. 

Is there something like this? 

Thanks

**Reply by davidjgraph** [reply]
There is no such documentation. We don't have the resources to write and maintain such a thing.
